src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.ts

```
@@ -1371,11 +1371,11 @@ export class AddEditPerDiemPage implements OnInit {
         } else {
           return forkJoin({
             orgSettings: this.orgSettingsService.get(),
-            orgUserSettings: this.orgUserSettingsService.get(),
+            employeeSettings: this.platformEmployeeSettingsService.get(),
           }).pipe(
-            map(({ orgSettings, orgUserSettings }) => {
+            map(({ orgSettings, employeeSettings }) => {
               if (orgSettings.projects.enabled) {
-                return orgUserSettings && orgUserSettings.preferences && orgUserSettings.preferences.default_project_id;
+                return employeeSettings && employeeSettings.default_project_id;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)